components/audio_video_preview/audio_video_preview.tsx

```diff
@@ -55,7 +55,11 @@ export default class AudioVideoPreview extends React.PureComponent<Props, State>
             video = document.createElement('video');
         }
 
-        const canPlayType = video.canPlayType(fileInfo.mime_type);
+        // rewriting the mimetype for quicktime to allow for playback in chrome browsers.
+        // mov files are encoded with MPEG-4 but the mimetype is set to quicktime.
+        const mimeType = (fileInfo.mime_type === 'video/quicktime') ? 'video/mp4' : fileInfo.mime_type;
+
+        const canPlayType = video.canPlayType(mimeType);
 
         this.setState({
             canPlay: canPlayType === 'probably' || canPlayType === 'maybe',
```

Comment:
While it was good, I prefer to add the exception directly, also added the bug from chromium in case it ever changes we can track it. 

This is not a blocker, and I'm approving, but just in case you prefer this version :)

```suggestion
        // mov files are encoded with MPEG-4 but the mimetype is set to quicktime, should be allowed in chrome
       // Long standing chrome bug: https://bugs.chromium.org/p/chromium/issues/detail?id=1120292
        const canPlayType = video.canPlayType(mimeType);

        this.setState({
            canPlay: canPlayType === 'probably' || canPlayType === 'maybe' || fileInfo.mime_type === 'video/quicktime',
```

Reply:
@Willyfrog - Would it be worth it to keep it the original way and allow the browser to decide if it can play? I didn't move it to the state because I wasn't sure the impact it might have by always assuming we can play `video/mp4` files.

Reply:
Thanks! I adjusted and added the comment.